.github/workflows/test.yml

```diff
@@ -229,8 +224,6 @@ jobs:
       fail-fast: false
       matrix:
         include:
-        - { name: 'GCC 5 (Ubuntu Xenial - 16.04)',    eval: 'CC=gcc-5 && CXX=g++-5',           build: 'release_strict' }
-        - { name: 'GCC 6 (Debian Stretch)',           eval: 'CC=gcc-6 && CXX=g++-6',           build: 'release_strict' }
```

Comment:
I think we need to be doing some 'release_strict' builds?

Reply:
The `Warnings` job sets `-DVTR_ENABLE_STRICT_COMPILE=on`, is that sufficient? Also, turns out `BUILD_TYPE=*_strict` doesn't actually do anything, see #1760.